crates/rust-analyzer/src/dispatch.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
     global_state::{GlobalState, GlobalStateSnapshot},
     main_loop::Task,
     version::version,
-    LspError, Result,
+    LspError,
```

Comment:
Seeing this refactor, I wonder if having a `replace use with qualified name` assist would be nice to have(reverse of `replace_qualified_name_with_use`)